cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,33 @@
 /// <reference types='cypress' />
 
+const firstName = 'Anderson';
+const lastName = 'Konstant';
+const userEmail = 'Anderson@gmail.com';
+const phonnumber = 802654594154;
+const Subjects = 'dsafgasfasd';
```

Comment:
The `Subjects` variable contains a placeholder value ('dsafgasfasd'). Ensure that this value is meaningful and relevant to the test case.